### ISSUE TITLE
Deep fryers never stop frying

### ISSUE THREAD

**Issue by sonime** [d3athrow/vgstation13#3607]
Yesterday I joined as a chef. I prayed and an admin turned on funfood. I deep-fried the wrapping paper and although the item popped out of the fryer when it was done, the fryer was still in its "frying" animation. Eventually it ran out of corn oil. When I used the second one, the same issue occured, but the server lagged horribly. I asked an admin to delete the fryers and that seemed to fix the lag.

tl;dr Fryers won't stop and it caused massive features.

**Comment by 9600bauds** [comment]
Happens since Pomf did that fix to stop the fryer from deleting the nukedisk.
